bigshout.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: future_fstrings -*-
+
+#    Friendly Telegram (telegram userbot)
+#    Copyright (C) 2018-2019 The Authors
+
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+
+from .. import loader, utils
+
+logger = logging.getLogger(__name__)
+
+
+def register(cb):
+    cb(ShoutOut())
+
+
+class ShoutOut(loader.Module):
+    """SHOUTS OUT A MESSAGE"""
+    def __init__(self):
+        self.name = _("Shout Out")
+
+    async def shoutoutcmd(self, message):
+        """Format: `.shoutout text`"""
+        args = utils.get_args(message)
+        if len(args) == 0:
+            await message.edit(_("Ehm... I can't shout nothing."))
+            return
+        messagestr = " ".join(args[0:])
+        text = " ".join(messagestr)
+        result = []
+        result.append(' '.join([s for s in text]))
```

Comment:
uhhhh. several points.

use " not '
[s for s in text] == list(text) == [*text], but you should really just use text, as any iterable works

Reply:
Also this is plain wrong i think?